extensions/wikia/VideoHandlers/handlers/MakerstudiosVideoHandler.class.php

```diff
@@ -10,6 +10,10 @@ public function getEmbed( $width, array $options = [] ) {
 		$height =  $this->getHeight( $width );
 		$sizeString = $this->getSizeString( $width, $height );
 		$url = $this->getEmbedUrl();
+		if ( !empty( $options['autoplay'] ) ) {
```

Comment:
I don't like that `autoplay` is not camelCase but that has nothing to do with this commit. 

LGTM!